distributed/bokeh/plots/status.py

```diff
@@ -0,0 +1,102 @@
+from bisect import bisect
+
+from bokeh.models import (
+    ColumnDataSource, Plot, Datetime, DataRange1d, Rect, LinearAxis,
+    DatetimeAxis, Grid, BasicTicker, HoverTool, BoxZoomTool, ResetTool,
+    PanTool, WheelZoomTool, Title
+)
+
+from distributed.utils import log_errors
+
+class Element(object):
+    """ Base class for Distributed UI Dashboard elements. """
+
+    def __init__(self):
+        """
+        Should be overridden by subclasses.
+
+        self.source ColumnDataSource
+        self.root should a layout-able object, a Plot, Layout, Widget or WidgetBox
+        """
+        self.source = None
+        self.root = None
```

Comment:
yes, ``root`` is the Bokeh term for a layout-able object, which encompasses plots, widgets and layouts (rows, columns and widgetboxes) of everything. I'll try to expand the docstring to explain this better and consider changing the property name.